src/generate/overrides.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+// Source data is not perfect, below are manual overrides applied to geo names
+const overrides = new Map([
+  // ['original', 'override']
+  ['Antigua And Barbuda', 'Antigua and Barbuda'],
+  ['Bosnia & Herzegovina', 'Bosnia and Herzegovina'],
+  ['Bolivia, Plurinational State Of', 'Bolivia'],
+  ['Bonaire, Saint Eustatius And Saba', 'Bonaire, Saint Eustatius and Saba'],
+  ['Democratic Republic Of Congo', 'Democratic Republic of Congo'],
+  ['Republic Of Congo', 'Republic of Congo'],
+  ['Czech Republic', 'Czechia'],
+  ['Ceuta, Mulilla', 'Ceuta and Mulilla'],
```

Comment:
Humm... Which is the original source? This should be Melilla, not ~Mulilla~! https://en.wikipedia.org/wiki/Melilla

Reply:
The original source (first string) seems to have been misspelled. Do you mind PRing this, since this one's already merged? Thank you!